README.md

```diff
@@ -177,6 +177,45 @@ $ mvn clean integration-test -Phit-refresh -Dport=8080
 
 Maven will start the server and you can see it at `http://localhost:8080`.
 
+## Using in servlet app
+
+Create a take with constructor accepting `ServletContext`:
+
+```java
+package com.myapp;
+
+public final class TkApp implements Take {
```

Comment:
@g4s8 And basically, any ``Take``, will be declared in ``web.xml`` as ``init`` param for ``SrvTake``, is that right?

Reply:
@amihaiemil HTTP method is a part of request head: `HTTP/1.1 GET /path`,
servlet parse it in request, but takes use `new RqMethod(req).method()` to read it, so I build back this http-head to be parsed in takes correctly (see RqHrom.HttpHead` class)